File: iraf/unix/boot/spp/rpp/ratlibf/upper.f
      subroutine upper (token)
      integer token (100)
      integer cupper
      integer i
      i = 1
23000 if (.not.(token (i) .ne. -2))goto 23002
      token (i) = cupper (token (i))
23001 i = i + 1
      goto 23000
23002 continue
      return
      end

File: unix/boot/spp/rpp/ratlibr/upper.r
include	defs

# upper - fold all alphas to upper case

   subroutine upper (token)
   character token (ARB)

   character cupper

   integer i

   for (i = 1; token (i) != EOS; i = i + 1)
      token (i) = cupper (token (i))

   return
   end
